apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -70,6 +71,40 @@ class OrderBlocksAnswerData(TypedDict):
     uuid: str
 
 
+class AnswerAttribs(TypedDict):
```

Comment:
Realize this is a bit inconsistent (so it might be worth refactoring everything to use this), but I think making this a NamedTuple (or dataclass) is better than a TypedDict. TypedDicts are better for situations where a dictionary is required (i.e. parts of a json payload), but the data here is only used internally. This will make other parts of the code easier to read.